servers/services/src/main/java/org/projectnessie/services/impl/BaseApiImpl.java

```diff
@@ -155,13 +161,21 @@ public CommitMeta squash(List<CommitMeta> metadata) {
                 .author(committer)
                 .authorTime(now);
         StringBuilder newMessage = new StringBuilder();
+
+        if (messageOverride != null) {
+          newMessage.append(messageOverride);
+        }
+
         Map<String, String> newProperties = new HashMap<>();
         for (CommitMeta commitMeta : metadata) {
           newProperties.putAll(commitMeta.getProperties());
```

Comment:
I think, we should skip this step for v2 as well, actually skipping the whole loop for v2, eventually changing the above `if` to 
```
if (messageOverride != null) { return newMeta.message(messageOverride).build(); }
```

Reply:
yes, but for now I'm going to merge just the message changes :)